src/client/java/aztech/modern_industrialization/machines/models/MachineBakedModel.java

```diff
@@ -57,22 +57,25 @@ public class MachineBakedModel implements IDynamicBakedModel {
     public static final String CASING_FOLDER = "machine_casing";
 
     public static ModelResourceLocation getCasingModelId(MachineCasing casing) {
-        return ModelResourceLocation.standalone(MI.id(CASING_FOLDER + "/" + casing.name));
+        return ModelResourceLocation
```

Comment:
Looks like a good use case for `casing.key.withPath(p -> CASING_FOLDER + "/" + p)`.